docs/guides/runtime-options-overview.mdx

```diff
@@ -50,18 +334,23 @@ options = Options()
 options.resilience_level = 1
 options.execution.shots = 2048
 ```
-## Pass options to a primitive
+<span id="pass-options"></span>
+## Set primitive options
+
+You can set options in an options class, when initializing the primitive, or both.
 
 ### Options class
 
-When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example:
+When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Specify the options in this format:  `primitive.options.option.sub-option.sub-sub-option = choice`.  For example: `estimator.options.dynamical_decoupling.enable = True`
```

Comment:
This section is a bit weird. It talks about creating and passing in `options` class, but the code doesn't actually show that (for both V1 and V2). I think the layout for "Set primitive options" should be

- Primitive initialization
    - you can pass in an instance of options class (`EstimatorOptions` and `SamplerOptions` for V2, `Options` for V1), or a dictionary when initializing Sampler/Estimator
- Update options after initialization
    - Both V1 and V2 support the syntax `<primitive>.options.<option>.<sub-option>.<sub-sub-option> = <choice>`, if you want to use auto complete
    - If you want to do bulk update, V1 supports `<primitive>.set_options()`, and V2 supports `<primitive>.options.update()`
- Run() method